decentraland/unity-renderer
# chore: remove skybox feature flag

Fixes #1775 

Remove procedural skybox feature flag, now it should be enabled all the time.
Remove the old `RenderProfile` implementation.

# How to test
The flag `DISABLE_PROCEDURAL_SKYBOX` shouldn't have any effect.
Check that avatar in the editor is not being affected by the skybox.
Check that all the settings for the skybox are still usable with the `DISABLE_PROCEDURAL_SKYBOX`
There shouldnt be any visual difference between this branch (with or without `DISABLE_PROCEDURAL_SKYBOX`)  and production
